bindings/azure/servicebusqueues/servicebusqueues.go

```diff
@@ -61,10 +64,36 @@ func (a *AzureServiceBusQueues) Init(metadata bindings.Metadata) error {
 	userAgent := "dapr-" + logger.DaprVersion
 	a.metadata = meta
 
-	ns, err := servicebus.NewNamespace(servicebus.NamespaceWithConnectionString(a.metadata.ConnectionString),
-		servicebus.NamespaceWithUserAgent(userAgent))
-	if err != nil {
-		return err
+	var ns *servicebus.Namespace
+	if a.metadata.ConnectionString != "" {
```

Comment:
Did you mean `if a.metadata.ConnectionString == ""` here? You want to use the Azure auth if not explicitly provided the ConnectionString, right?

Also, it might be clearer to just say `a.metadata.NamespaceName != ""` since `parseMetadata` now enforces that the ConnectionString should not be set if NamespaceName is also set.